chapter1/Voting.sol

```diff
@@ -19,12 +19,12 @@ contract Voting {
   }
 
   function totalVotesFor(bytes32 candidate) returns (uint8) {
-    if (validCandidate(candidate) == false) throw;
```

Comment:
So instead of 

    function totalVotesFor(bytes32 candidate) returns (uint8) {
    if (validCandidate(candidate) == false) revert();
    return votesReceived[candidate];
  }

You would have:

    function totalVotesFor(bytes32 candidate) returns (uint8) {
        require(validCandidate(candidate));
        return votesReceived[candidate];
      }

Am I correct?